addon/middleware/index.js

```diff
@@ -0,0 +1,13 @@
+//only here as a vanilla pass thru so we can
```

Comment:
because the addon/services/redux file calls out an explicit "import" for middleware I need this file present. If I include the `redux-thunk` version here (instead of app) the end user is then required to have redux-thunk installed even if they opt-out and use redux- saga instead